dataprofiler/profilers/histogram_utils.py

```diff
@@ -2,329 +2,11 @@
 https://github.com/numpy/numpy/blob/v1.19.0/numpy/lib/histograms.py
 Histogram-related functions
 """
-import functools
```

Comment:
exact same code as in numpy, we didn't need to copy and could just import instead as seen at bottom of the changes